File: Task2/src/HospitalityHub.Core/DTOs/Room/UpdateRoomRequest.cs
using HospitalityHub.Core.Enums;

namespace HospitalityHub.Core.DTOs.Room;

public record UpdateRoomRequest(
    int Id,
    int Number,
    ERoomStatus Status,
    decimal BasePrice,
    decimal DiscountPercent,
    int Stage,
    ERoomType RoomType,
    int HotelId
);

File: Task4/hospitality-hub/src/app/api-proxy/models/update-room-request.ts
/* tslint:disable */
/* eslint-disable */
import { ERoomStatus } from '../models/e-room-status';
import { ERoomType } from '../models/e-room-type';
export interface UpdateRoomRequest {
  apiKey?: string | null;
  baseLockUri?: string | null;
  basePrice?: number;
  discountPercent?: number;
  hotelId?: number;
  id?: number;
  number?: number;
  roomType?: ERoomType;
  stage?: number;
  status?: ERoomStatus;
}
